eventhandling/configureEvent.go

```diff
@@ -164,6 +164,9 @@ func (eh ConfigureMonitoringEventHandler) updatePrometheusConfigMap(eventData ke
 
 	cmPrometheus, err := api.CoreV1().ConfigMaps(env.PrometheusNamespace).Get(context.TODO(), env.PrometheusConfigMap, metav1.GetOptions{})
 	if err != nil {
+		if strings.Contains(err.Error(), "is forbidden") {
```

Comment:
This might be too generic - can we extend this, either using a Regex or glob-like check?
```
configmaps (*) is forbidden: User (*) cannot get resource (*) in API group (*) in the namespace (*)
```
Or maybe add some more strings.Contains, as an easy workaround?

Reply:
Matched it using a regex expression. Let me know if this is sufficient.

Reply:
Should be resolved now.